packages/p2p-v2/src/routes/Router.tsx

```diff
@@ -7,24 +7,30 @@ const prefix = '/cashier/p2p-v2';
 type TRoutes = `${typeof prefix}/cashier/p2p-v2` | `${typeof prefix}`;
 
 declare module 'react-router-dom' {
-    export function useHistory(): { push: (path: TRoutes | string) => void };
+    export function useHistory(): {
+        push: (path: TRoutes | string) => void;
+        replace(arg0: { pathname: string; search: string }): void;
+    };
 
     export function useRouteMatch(path: TRoutes): boolean;
 }
 
 const Router: React.FC = () => {
     const history = useHistory();
+
     if (history.location.pathname === prefix) {
         history.push(`${prefix}/${routesConfiguration[0].path}`);
     }
     return (
-        <Switch>
-            {routesConfiguration.map(({ Component, path }) => (
-                <Route key={path} path={path}>
-                    {Component}
-                </Route>
-            ))}
-        </Switch>
+        <>
+            <Switch>
+                {routesConfiguration.map(({ Component, path }) => (
+                    <Route key={path} path={path}>
+                        {Component}
+                    </Route>
+                ))}
+            </Switch>
+        </>
```

Comment:
do you need these empty tags?

```suggestion
            <Switch>
                {routesConfiguration.map(({ Component, path }) => (
                    <Route key={path} path={path}>
                        {Component}
                    </Route>
                ))}
            </Switch>
```

Reply:
spooky prettier 👀